packages/console/src/pages/TenantSettings/Subscription/PlanComparisonTable/renderers/BetaFeatureFlag.tsx

```diff
@@ -0,0 +1,37 @@
+import { cond, type Optional } from '@silverhand/essentials';
+
+import DynamicT from '@/ds-components/DynamicT';
+
+import QuotaValueWrapper from './QuotaValueWrapper';
+
+type Props = {
+  isEnabled: Optional<boolean>;
```

Comment:
nit: `isEnabled?: boolean` for simplicity

Reply:
Updated.